src/builder/create_embed.rs

```diff
@@ -237,7 +237,7 @@ impl CreateEmbed {
     ///     }
     /// }
     ///
-    /// let mut client = Client::builder("token").event_handler(Handler).await?;
+    /// let mut client = Client::builder("token", Default::default()).event_handler(Handler).await?;
```

Comment:
It would be better to use `GatewayIntents` here to make the required parameter type clear.
```suggestion
    /// let mut client = Client::builder("token", GatewayIntents::default()).event_handler(Handler).await?;
```

Reply:
Thank you